### ISSUE TITLE
pretrained weight

### ISSUE THREAD

**Issue by dpogosov** [wvangansbeke/LaneDetection_End2End#30]
Hi Wouter, could you please provide me with the pre-trained model weights; or even share it? It would be a good hot-start. Thank you in advance!

**Comment by wvangansbeke** [comment]
Hi @dpogosov,

You mean a pretrained network trained with segmentation ground truth and cross-entropy loss? You can easily train it since I also provided the ground segmentation maps. I don't think I have a pretrained network lying around atm.

Best,
Wouter
